### PR TITLE
34981 Saving lists or forms without a name

### PR DESCRIPTION
**Pull request by yeneastgate**
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/527

changed log :
add notice when saving lists or forms without a name.

### CODE REVIEW

File: plugin/Gui/AdminPageAddressListSettings.php [hunk 1]
```diff
@@ -366,7 +380,7 @@ public function doExtraEnqueues()
 	protected function checkFixedValues($row)
 	{
 		$table = RecordManager::TABLENAME_LIST_VIEW_ADDRESS;
-		$result = isset($row[$table]['name']) && $row[$table]['name'] != null;
+		$result = isset($row[$table]['name']) && $row[$table]['name'] != null && !empty(trim($row[$table]['name']));
```
Comment (andernath) [inline plugin/Gui/AdminPageAddressListSettings.php:383]: $row[$table]['name'] != null 
Is this necessary or is not null already checked with isset?

Same question for other classes below.

**Review by andernath — COMMENTED** [review]
Finished review and have one question. Please check. @yeneastgate :)

File: plugin/Gui/AdminPageAddressListSettings.php (hunk 1 above)
Comment (yeneastgate) [reply plugin/Gui/AdminPageAddressListSettings.php:383]: @andernath I will check and let you know soon.

Comment (yeneastgate) [reply plugin/Gui/AdminPageAddressListSettings.php:383]: @andernath In this case, the isset function already checks undefined and null => so $row[$table]['name'] != null is not needed.
I will check and delete it

**Review by andernath — APPROVED** [review]

**Pull request merged**